### PR TITLE
Fix #6442 Remove IRC

### LINKED ISSUE #6442: Remove IRC from Contribution Guide
Remove references to the legacy IRC channel from the contribution guides in 9,10 and 11. We no longer regularly use them.

### PR DESCRIPTION
**Pull request by gregw**
Fix #6442 Remove IRC from contributing documentation

### CODE REVIEW

**Review by lachlan-roberts — DISMISSED** [review]
There's also some comments about IRC on line 18. Might want to change this as well.

This is also in `old_docs` in jetty-10 but not in the actual `contribution-guide`.

File: jetty-documentation/src/main/asciidoc/reference/contributing/community.adoc [hunk 1]
```diff
@@ -20,8 +20,9 @@
 === Community
 
 Developers and users alike are welcome to engage the Jetty community.
-We all have day jobs here so don't just ask questions on IRC and frustrated if no one answers right away.
-Stick around to hear an answer, one is likely coming at some point!
+We all have day jobs here so don't just ask questions and get frustrated if 
+no one answers right away. Stick around to hear an answer, one is 
+likely coming at some point!
```
Comment (lachlan-roberts) [inline jetty-documentation/src/main/asciidoc/reference/contributing/community.adoc:25]: I don't think you're supposed to put line breaks in like this.

Comment (joakime) [reply jetty-documentation/src/main/asciidoc/reference/contributing/community.adoc:25]: The asciidoc will merge them together.

Don't forget, the asciidoc is supposed to be readable on it's own, before being processed into html/pdf.

**Review by joakime — APPROVED** [review]

File: jetty-documentation/src/main/asciidoc/reference/contributing/community.adoc (hunk 1 above)
Comment (lachlan-roberts) [reply jetty-documentation/src/main/asciidoc/reference/contributing/community.adoc:25]: I was doing this before but told not to by Simone or Chris. The rest of the documentation uses only 1 line per sentence, and to view it in the IDE you use word wrap.

**Pull request merged**